blockly-core/core/ui/block.js

```diff
@@ -1454,6 +1472,69 @@ Blockly.Block.prototype.setMovable = function(movable) {
 };
 
 /**
+ * Get whether this block represents a toolbox block that can only be
+ * used a limited number of times
+ * @return {boolean}
+ */
+Blockly.Block.prototype.hasLimit = function() {
+  return (this.isInFlyout || Blockly.editBlocks) && this.limit_ !== undefined;
```

Comment:
What does the `Blockly.editBlocks` part of this do? Would there be a way to name & extract the `(this.isInFlyout || Blockly.editBlocks)` clause of this? e.g. `var shouldDisplayLimits = this.isInFlyout || Blockly.editBlocks;` (if that's what it means)